polkadot/xcm/xcm-builder/src/location_conversion.rs

```diff
@@ -936,4 +979,22 @@ mod tests {
 		};
 		assert!(ForeignChainAliasAccount::<[u8; 32]>::convert_location(&mul).is_none());
 	}
+
+	#[test]
+	fn remote_account_convert_on_para_sending_relay_treasury() {
```

Comment:
probably I am missing some context here, I can understand `remote_account_convert_on_para` - that this converter will be used as a part of `LocationToAccountId` converter on parachain,
but what does exactly mean `sending_relay_treasury`?

Reply:
its a badly named test i think, but its meant to be remote treasury on a parachain to another parachain

Reply:
hmm, because from the **parachain's point of view** this:
``` 
        MultiLocation {
			parents: 1,
			interior: X1(Plurality { id: BodyId::Treasury, part: BodyPart::Voice }),
		};
```
represents Plurality-Treasury-Voice location on relay chain (because `parents: 1`).


From the **parachain's point-of-view** a **relay chain plurality** looks like:
```
MultiLocation { parents: 1, interior: X1(Plurality { .. }
```

From the **parachain's point-of-view** an **other parachain plurality** looks like:
```
MultiLocation { parents: 1, interior: X2(Parachain(other_parachain_para_id), Plurality { .. }
```

So I would suggest to cover at least these two case in this test, to be sure that it works as expected.
Ideally, it should maybe cover more cases:
DescribeTreasuryVoiceTerminal is deployed on relay chain and local treasury location.
DescribeTreasuryVoiceTerminal is deployed on relay chain and  parachain treasury location? (not sure about this case)
DescribeTreasuryVoiceTerminal is deployed on parachain and local treasury location.
DescribeTreasuryVoiceTerminal is deployed on parachain and relay chain treasury location.
DescribeTreasuryVoiceTerminal is deployed on parachain and other parachain treasury location.

(but as I said, I dont have full context here)

Reply:
Hi @bkontur thanks for this, I've renamed the tests and added another test to cover the para to para treasury conversion



I _think_ the `DescribeFamily` covers all the other cases but please correct me if I am mistaken.